app/src/main/java/com/stl/skipthelibrary/AddBooksActivity.java

```diff
@@ -0,0 +1,87 @@
+package com.stl.skipthelibrary;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+public class AddBooksActivity extends AppCompatActivity {
+    Context mContext;
```

Comment:
General question, why is it common to see mContext instead of just context as the variable name? Does m represent my?

Reply:
I believe mContext stands for myContext? You see it used a lot in Android tutorials and best practice guides